src/App.js

```diff
@@ -85,13 +85,17 @@ class App extends Component {
         <div id="controllers">
           <div id="total-display" className="row">
             <div className="col-8" style={{padding: 0}}>
-              <button style={{float: "left"}} onClick={() => {this.setState({mod: this.state.mod - 1})}}>
+              <button className="mod-minus" onClick={() => {
+                  this.setState({mod: this.state.mod - 1})
+                }}>
                 -
               </button>
-              {this.renderDesc()}
-              <button style={{float: "right"}} onClick={() => {this.setState({mod: this.state.mod + 1})}}>
+              <button className="mod-plus" onClick={() => {
```

Comment:
This is a great fix, but the word Mod doesn't seem to be very clear. Maybe use an alternate word like difference, modifier, scaler, or something else, because mod can be a modulator, moderator, module, and a bunch of other terms.